test-apps/ui-test-app/src/frontend/appui/AppUi.tsx

```diff
@@ -386,6 +387,10 @@ export class AppUi {
             key: "l",
             item: AccuDrawPopupTools.showHTMLElement,
           },
+          {

```

Comment:
Should all these API calls be turned into one or more Immediate Tools so that can be activated via key-in? This would allow keyboard shortcuts just to hold key-ins (and arguments) to parse and run.

Reply:
Possibly, but for another PR (?)